experiment/telegram/telegram_otherwise.go

```diff
@@ -0,0 +1,206 @@
+// +build !cgo
```

Comment:
Maybe the title of this file can be `telegram_native.go`?

Reply:
It's named liked that because the other file is named `telegram_cgo.go`. I think the naming here does not matter much, since we'll remove the MK implementation and change the name of these files as soon as we all agree that this new implementation is WAI.

Reply:
https://github.com/ooni/probe-engine/issues/112